src/js/view/controls/controlbar.js

```diff
@@ -182,9 +189,7 @@ export default class Controlbar {
             cast: createCastButton(() => {
                 _api.castToggle();
             }, localization),
-            fullscreen: button('jw-icon-fullscreen', () => {
-                _api.setFullscreen();
-            }, localization.fullscreen, cloneIcons('fullscreen-off,fullscreen-on')),
+            fullscreen: fullscreenButton,
```

Comment:
What's the point of having a `fullscreenButton` var if it only gets used once ?

Reply:
@karimMourra  - I wanted to keep the code clean. It is also what is done with other buttons (i.e. `muteButton` & `nextButton`).

Reply:
sure